src/ctap/test_helpers/env.rs

```diff
@@ -0,0 +1,49 @@
+use crate::clock::CtapInstant;
+use crate::ctap::command::{
+    AuthenticatorAttestationMaterial, AuthenticatorConfigParameters,
+    AuthenticatorVendorConfigureParameters, Command,
+};
+use crate::ctap::data_formats::ConfigSubCommand;
+use crate::ctap::status_code::Ctap2StatusCode;
+use crate::ctap::{key_material, Channel, CtapState};
+use crate::env::Env;
+
+// In tests where we define a dummy user-presence check that immediately returns, the channel
+// ID is irrelevant, so we pass this (dummy but valid) value.
+const DUMMY_CHANNEL: Channel = Channel::MainHid([0x12, 0x34, 0x56, 0x78]);
+
+pub struct SetupEnterpriseAttestationResponse {
+    pub attestation_material: AuthenticatorAttestationMaterial,
+}
+
+pub fn setup_enterprise_attestation(
+    state: &mut CtapState,
+    env: &mut impl Env,
+) -> Result<SetupEnterpriseAttestationResponse, Ctap2StatusCode> {
+    let config_params = AuthenticatorConfigParameters {
```

Comment:
Let's move this right before it's used, to not extend scope unnecessarily and improve readability.

Reply:
done